src/components/Onboarding/Steps/Step.js

```diff
@@ -0,0 +1,44 @@
+import React, { useMemo } from 'react'
+import StepItem from './StepsItem'
+import { useOnboardingState } from '@providers/Onboarding'
+
+function Step({ currentStep, label, step, stepNumber, substeps }) {
+  const { step: onboardingStep } = useOnboardingState()
+  const [displayedSubsteps] = useMemo(() => {
+    const displayedSubsteps = substeps.map((step, index) => {
+      if (substeps.length > 0) {
+        const label = step[0].title
+
+        const statusIndex = step[1]
```

Comment:
Trying to understand the a bit the logic here, based on the naming what should be the statusIndex variable since is the value `index - hiddenCount` right? what status is that indicating?

Reply:
i think we just miss this small comment, just to understand